packages/components/alert/src/alert.ts

```diff
@@ -37,6 +37,11 @@ export const alertProps = buildProps({
     required: false,
     default: undefined,
   },
+  closable: {
```

Comment:
you have other components having this property and it's named "isClosable".
Which makes sense as it is a boolean.
And you wish to be coherent amongst the whole lib when naming things ;)